ArduCopter/Parameters.h

```diff
@@ -447,6 +447,10 @@ class Parameters {
     AP_Int16        gcs_pid_mask;
 
 #if MODE_THROW_ENABLED == ENABLED
+    enum class THROW_MOTOR_START : uint8_t {
```

Comment:
Argh.  Sorry, I neglected to suggest changing the class name; we tend to use mixed-caps for the enum-class name:

```suggestion
    enum class ThrowMotorStart : uint8_t {
```